sdk/search/azure-search-documents/swagger/readme.md

```diff
@@ -428,4 +428,25 @@ directive:
       transform: >-
         return $
         .replace(/public Boolean isShouldDetectOrientation\(\) \{/g, "public Boolean shouldDetectOrientation() {")
+
+    # Changed isRetrievable to isHidden
+    - from: swagger-document
+      where: $.definitions.Field.properties
+      transform: >
+         $.hidden = $.retrievable;
+         $.hidden = {
+            "type": "boolean", 
+            "description": "A value indicating whether the field will be returned in a search result. This property must be true for key fields, and must be null for complex fields. You can hide a field from search results if you want to use it only as a filter, for sorting, or for scoring. This property can also be changed on existing fields and enabling it does not cause an increase in index storage requirements."
```

Comment:
Should update this so the Javadocs in the class generate with this must be false for key fields.

Reply:
I have merged your comments but regen the code. That's why the merge changes are gone. Done with transformation change